TeamCode/src/main/java/org/firstinspires/ftc/teamcode/movement/MovementMotors.java

```diff
@@ -0,0 +1,41 @@
+package org.firstinspires.ftc.teamcode.movement;
+
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class MovementMotors {
+    private DcMotor fl, fr, bl, br;
+    private Telemetry console;
+
+    public MovementMotors(HardwareMap hardwareMap, Telemetry telemetry) {
+
+        console = telemetry;
```

Comment:
Daca la restul ai folosit `this.*`, ar face sens sa faci si aici la fel: `this.console`. Vezi si mai jos.